src/windows.toit

```diff
@@ -143,26 +174,62 @@ is-absolute "../foo/bar"       // False.
 ```
 */
 is-absolute path/string -> bool:
+  return is-absolute-volume path or is-absolute-unc path
+
+/**
+Path starts with a UNC volume.
+For example: `//host/share`, `//./UNC/host/share`, ...
+*/
+is-absolute-unc path/string -> bool:
```

Comment:
Unless really needed, I would remove this and the `is-absolute-volume` function.

Reply:
is-absolute-volume is needed.